### PR TITLE
DDP-5816 family history: indicate all questions were completed

### PR DESCRIPTION
**Pull request by aglushko-broad**

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.html [hunk 1]
```diff
@@ -11,9 +11,18 @@
 
             <div *ngIf="instance.statusCode !== 'CREATED'"
                  class="modal-activity-block__success-message">
-                <mat-icon inline>check_circle</mat-icon>
-                <span [innerHTML]="'SDK.ModalActivityBlock.NumQuestionsAnswered'
-          | translate: { number: instance.numQuestionsAnswered }"></span>
+                <mat-icon *ngIf="instance.numQuestionsAnswered === instance.numQuestions" inline>check_circle</mat-icon>
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.html]: **_As an option_**, keep this check as a boolean getter in the component, like e.g. `isAllQuestionsCompleted`
and also use it in the next line

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  let component: TestHostComponent;
+  let fixture: ComponentFixture<TestHostComponent>;
+  let debugElement: DebugElement;
+
+  const successMessageSelector = '.modal-activity-block__success-message';
+  const successMessageIconSelector = `${successMessageSelector} mat-icon`;
+  const successMessageLabelSelector = '.modal-activity-block__label';
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.spec.ts:46]: Nice tests!

**Review by alextanlan — DISMISSED** [review]

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.html (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.html]: good idea, done!

**Review by alextanlan — APPROVED** [review]

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.spec.ts (hunk 2 above)
Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.spec.ts:46]: Oh Yeah!
592bc6a592bbacc67ef9871690768996

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**